elixir/plug/app/lib/plug_example.ex

```diff
@@ -18,4 +18,11 @@ defmodule PlugExample do
   get "/error" do
     raise "Whoops!"
   end
+
+  get "/custom" do
+    Appsignal.Tracer.root_span()
+    |> Appsignal.Span.set_sample_data("params", %{"wow_such_custom" => "amaze"})
```

Comment:
Nit: if you could copy the stuff from the Phoenix example here it tests a lot more: https://github.com/appsignal/test-setups/blob/d2e4d55ca636f9ab51e64d41ac645ced11edac21/elixir/phoenix-example/app/lib/appsignal_phoenix_example_web/controllers/page_controller.ex#L66-L118